optuna/integration/dask.py

```diff
@@ -424,10 +424,18 @@ class DaskStorage(BaseStorage):
     Optuna storage object.
 
     See `this example <https://github.com/optuna/optuna-examples/blob/master/
-    dask/dask_simple.py>`__
+    dask/dask_simple.py>`_ or the following YouTube video
     for how to use :obj:`DaskStorage` to extend Optuna's in-memory storage class to run across
     multiple processes.
 
+    .. raw:: html
+
+       <iframe width="560" height="315" src="https://www.youtube-nocookie.com/embed/euT6_h7iIBA"
```

Comment:
[Notes] I confirmed that the embedded video did not use cookies. Thank you for using `youtube-nocookie`.

c.f., https://github.com/optuna/optuna/blob/30fabb90fafbb499536e5999f4caec2ffe0c2963/docs/source/tutorial/index.rst?plain=1#L10